tests/unit_test.py

```diff
@@ -13,40 +13,53 @@
 import os
 import pytest
 
-@pytest.mark.usefixtures("requests")
+from resources.User_request import dtrequest
+from run import create_app
+import config
+
+
+@pytest.mark.unittest
 class ApiTest(unittest.TestCase):
     """ API endpoints test case"""
 
+    def setUp(self):
+        """Declare test variables and initialize app."""
+        self.app = create_app(config_filename="testing")
+        self.client = self.app.test_client
+        self.status = {'status': 'Active'}
+
+        
+
     def test_api_can_get_all_requests(self):
         """Test api Get all the requests for a logged in user"""
-        res = self.client().post('/api/v1/user/requests/', requests)
+        res = self.client().post('/api/v1/user/requests/', dtrequest)
         self.assertEqual(res.status_code, 201)
         res = self.client().get('/api/v1/user/requests')
         self.assertEqual(res.status_code, 200)
         self.assertIn("John doe", str(res))
 
     def test_api_can_get_request_by_id(self):
         """Test api can get a request for a logged in user"""
-        res = self.client().post('/api/v1/user/requests/', requests)
+        res = self.client().post('/api/v1/user/requests/', dtrequest)
         self.assertEqual(res.status_code, 201)
-        json_res = json.loads()
+        json_res = json.loads(dtrequest)
         result = self.client().get(
-            '/api/v1/user/requests{}'.format(json_res['id']))
+            '/api/v1/user/requests{}'.format(json_res['status']))
         self.assertEqual(result.status_code, 200)
-        self.assertIn("John doe")
+        self.assertIn("Resolved", dtrequest)
 
     def test_api_request_can_be_modified(self):
         """Test api can modify a request"""
-        res = self.client().post('/api/v1/user/requests/', data={"user":"john doe"})
-        res = self.client().put('/api/v1/user/requests/1', data ={"user":"John sue doe"})
+        res = self.client().post('/api/v1/user/requests/1', data={"status":"Approved"})
```

Comment:
Does this test work? I see you are creating a new request to the request with id 1 and the payload just has `{"status":"Approved"}` as opposed to creating a new request and the using a put request to edit the status to `resolved`

Reply:
I saw the error later, its already fixed, I'll push it asap